CHANGELOG.md

```diff
@@ -15,8 +15,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - Added new cloud region codes: hsg, abl, dfw, pbv, nbq, ibg, pcz, mez, den, kal
 - Added rowMetadata support, new API for Get/Put/Delete/MultiDelete request and result
   get/set RomMetadata.
-- Added row creation time support, new API: GetResult getCreationTime(), 
+- Added row creation time support, new API: GetResult getCreationTime(),
   Put/Delete/Write/WriteMultiple Result getExistingCreationTime().
+- Added an optional timeout for the login timeout used by
+  StoreAccessTokenProvider when logging into a store on premises. Previously
+  this timeout was hard-coded to 30s. The new default value is 5s
```

Comment:
Will this break legacy apps that somehow depend on the 30 second timeout? Should the default be 30 seconds to maintain backward compatibility?

Reply:
Seems unlikely given that it was pretty arbitrary in the first place, but one thing we should discuss is to use the request timeout when it's available and that'd solve this problem